packages/audio-element/index.js

```diff
@@ -1,3 +1,4 @@
 import AudioElement from './lib/audio-element.js';
+import AudioSource from './lib/models/audio-source.js';
 
-export default AudioElement;
+export { AudioElement, AudioSource };
```

Comment:
question: does eslint bark at this?

Reply:
Not that I've seen. What warning are you thinking it'd be throwing?

This file just acts as the module entry point so when you do `import { AudioElement } from @internetarchive/audio-element`, it knows where to find the actual class you're looking for.